src/app/pages/plans/plans.page.css

```diff
@@ -8,6 +8,8 @@
   margin: 0 auto;
   margin-bottom: var(--spacing-large);
   overflow: auto;
+  padding-left: var(--spacing-small);
```

Comment:
jeg tror du kan gjerne gå for spacing-medium her. Også hvis du vil du kan skrive det på en line bare med 
```css
padding: 0px var(--spacing-medium);
```